app/controllers/state_file/questions/terms_and_conditions_controller.rb

```diff
@@ -2,7 +2,14 @@ module StateFile
   module Questions
     class TermsAndConditionsController < QuestionsController
       def edit
-        @li_items = I18n.t('state_file.questions.terms_and_conditions.edit.list_items_html', privacy_policy_link: state_file_privacy_policy_path).split('</li>')
+        ownership = current_state_code == "nj" ?
+          "#{I18n.t('general.code_for_america')} #{I18n.t('general.and')} #{I18n.t('general.nj_ooi')}" :
```

Comment:
[dust/pebble] Similar to conversations we've had previously, it feels more correct for us to track a single key for "owner" that says either "Code for America" or "Code for America and New Jersey Office of Innovation" rather than keeping them as two keys joined by an "and" translation. I think changing it will be more resilient across languages so that we're not coding assumptions about grammatical structure but can instead let the language keys handle that.

Reply:
oh thanks - yes it felt awkward to construct the string the way i did, but no better way came to mind. i'll do it that way...

Reply:
now updated to use a single translation key with an `nj` option and a `default`